xml/admin_gui_dashboard.xml

```diff
@@ -392,123 +392,123 @@
-        The sum of clients' read and write operations per second.
+        Shows the ratio of used and available raw storage capacity.
        </para>
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term><guimenu>Client Read/Write</guimenu></term>
+      <term><guimenu>Objects</guimenu></term>
       <listitem>
        <para>
-        Visualizes clients' read/write ratio.
+        Shows the number of data objects stored in the cluster.
        </para>
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term><guimenu>Recovery Throughput</guimenu></term>
+      <term><guimenu>PGs per OSD</guimenu></term>
       <listitem>
        <para>
-        The throughput of data recovered per second.
+        Shows the number of placement groups per OSD.
        </para>
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term><guimenu>Scrub</guimenu></term>
+      <term><guimenu>PG Status</guimenu></term>
       <listitem>
        <para>
-        Shows the scrub (see <xref linkend="op-pg-scrubpg"/>) status. It is
-        either <option>inactive</option>, <option>enabled</option>, or
-        <option>active</option>.
+        Displays a chart of the placement groups according to their status.
        </para>
       </listitem>
      </varlistentry>
     </variablelist>
    </sect3>
-   <sect3 xml:id="dashboard-widgets-capacity">
-    <title>Capacity Widgets</title>
+   <sect3 xml:id="dashboard-widgets-performance">
+    <title>Performance Widgets</title>
     <para>
-     <guimenu>Capacity</guimenu> widgets show brief information about the
-     storage capacity.
+     <guimenu>Performance</guimenu> widgets refer to basic performance data of
+     &ceph; clients.
     </para>
     <figure>
-     <title>Capacity Widgets</title>
+     <title>performance Widgets</title>
      <mediaobject>
       <imageobject role="fo">
-       <imagedata fileref="dashboard_widgets_capacity.png" width="100%" format="PNG"/>
+       <imagedata fileref="dashboard_widgets_perf.png" width="100%" format="PNG"/>
       </imageobject>
       <imageobject role="html">
-       <imagedata fileref="dashboard_widgets_capacity.png" width="100%" format="PNG"/>
+       <imagedata fileref="dashboard_widgets_perf.png" width="100%" format="PNG"/>
       </imageobject>
      </mediaobject>
     </figure>
     <variablelist>
      <varlistentry>
-      <term><guimenu>Pools</guimenu></term>
+      <term><guimenu>Client IOPS</guimenu></term>
       <listitem>
        <para>
-        Shows the number of pools in the cluster.
+        The amount of clients' read and write operations per second.
        </para>
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term><guimenu>Raw Capacity</guimenu></term>
+      <term><guimenu>Client Throughput</guimenu></term>
       <listitem>
        <para>
-        Shows the ratio of used and available raw storage capacity.
+        The sum of clients' read and write operations per second.
```

Comment:
(Not incurred by this PR). The definition here sounds like IOPS.

Reply:
I'm not sure what you need to be changed here?

Reply:
The original description was wrong already. "The sum of clients' read and write operations per second." was used for both the description of the "Client IOPS" and "Client Throughput" widget.

The description for Client Throughput should read:
```suggestion
        The amount of data transferred to/from Ceph clients in multiples of bytes per second.
```